src/main/java/wooteco/subway/admin/controller/LineController.java

```diff
@@ -3,63 +3,70 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import wooteco.subway.admin.domain.Line;
-import wooteco.subway.admin.dto.LineDetailResponse;
-import wooteco.subway.admin.dto.LineRequest;
-import wooteco.subway.admin.dto.LineResponse;
-import wooteco.subway.admin.dto.LineStationCreateRequest;
+import wooteco.subway.admin.dto.request.LineRequest;
+import wooteco.subway.admin.dto.request.LineStationCreateRequest;
+import wooteco.subway.admin.dto.response.LineDetailResponse;
+import wooteco.subway.admin.dto.response.LineResponse;
+import wooteco.subway.admin.dto.response.WholeSubwayResponse;
 import wooteco.subway.admin.service.LineService;
 
 import java.net.URI;
 import java.util.List;
 
 @RestController
 public class LineController {
-    private LineService lineService;
+	private LineService lineService;
 
-    public LineController(LineService lineService) {
-        this.lineService = lineService;
-    }
+	public LineController(LineService lineService) {
+		this.lineService = lineService;
+	}
 
-    @PostMapping(value = "/lines")
-    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest view) {
-        Line persistLine = lineService.save(view.toLine());
+	@PostMapping(value = "/lines")
+	public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest view) {
+		Line persistLine = lineService.save(view.toLine());
 
-        return ResponseEntity
-                .created(URI.create("/lines/" + persistLine.getId()))
-                .body(LineResponse.of(persistLine));
-    }
+		return ResponseEntity
+				.created(URI.create("/lines/" + persistLine.getId()))
+				.body(LineResponse.of(persistLine));
+	}
 
-    @GetMapping("/lines")
-    public ResponseEntity<List<LineResponse>> showLine() {
-        return ResponseEntity.ok().body(LineResponse.listOf(lineService.showLines()));
-    }
+	@GetMapping("/lines")
+	public ResponseEntity<List<LineResponse>> showLines() {
+		return ResponseEntity.ok().body(LineResponse.listOf(lineService.showLines()));
+	}
 
-    @GetMapping("/lines/{id}")
-    public ResponseEntity<LineDetailResponse> retrieveLine(@PathVariable Long id) {
-        return ResponseEntity.ok().body(lineService.findLineWithStationsById(id));
-    }
+	@GetMapping("/lines/detail")
+	public ResponseEntity wholeLines() {
```

Comment:
제네릭 타입을 명시적으로 표시 해주어야 Type safe 합니다 ㅎㅎ
또한 다른 개발자가 해당 컨트롤러만 봤을때 어떤 응답을 하고 있구나 한눈에 알 수 있어요~


```suggestion
	public ResponseEntity<WholeSubwayResponse> wholeLines() {
```

Reply:
중간중간 빠진 부분이 있네요..😂

저번 미션에서 화투에게 리뷰 받았던 부분인데 아직 체화가 되지 않았나봅니다!

의식적인 연습으로 체화해보도록하겠습니다.

좋은 코멘트 감사드려요😌